index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    <div class="container">
+    
+    <header class="portfolio-header">
+    <div class="logo">My Logo</div>
+    
+        <div class="menu-mobile">
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+    </div>
+
+    </header> 
+  
+    
+
+<h1 class="salutation">
+   Hello. I’m Austin.
+    Glad to see you!</h1>
+<p class="developer">
+    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h2 class="connect">LET'S CONNECT</h2>
+
+<div class="social">
+    <a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a>
+    <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a>
+    <a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a>
+    <a href="https://github.com/ausiemass/portfolio"><img src="images/Vector (1).jpg" alt="GITHUB"></a>
+    <a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a>
+   
+</div>
```

Comment:
- There are two issues I have mentioned below. Kindly fix those.
- [ ] Here the social icons are `list of items`. A `list of items` should be wrapped by `ul` and a specific `icon or item` should be wrapped by `li` as it is the best semantic practice of HTML.
  - Here is an example: 
  - `<ul>`
  - `  <li><a>icon1<a></li>`
  - `  <li><a>icon2<a></li>`
  - `<ul>`
- [ ] The images you have used here are not `displaying` as there is no `image` folder in your GitHub repository.
- ![image](https://github.com/ausiemass/portfolio-austin/assets/60083437/9a2ad4d9-dfbf-4901-9d6b-08a9eb9809a6)